content/index.md

```diff
@@ -17,70 +17,296 @@ footer:
+                  {% endif %}
+                </a>
+              </li>
+            {% endif %}
+          {% endfor %}
+        </ul>
+      {% endif %}
+    {% endif %}
+  </li>
+  {% endfor %}
 </ul>
+{% else %}
+<p><em>No ACT Rules available for this criterion yet.</em></p>
+{% endif %}
+</div>
+{% endif %}
+{% comment %}End Principles, guidelines, and criteria{% endcomment %}
+    {% endfor %}
+<p class="back-to-top-link"><a href="#top">Back to top</a></p> 
+  {% endfor %}
+{% endfor %}
 
-## Deprecated ACT Rules
 
-The ACT Rules below are deprecated and are no longer maintained. For details on why a rule was deprecated see the "Deprecated" section at the top of the rule.
+{%- comment -%}
+  Collect all unique non-WCAG requirement titles and keys, and sort by title
+{%- endcomment -%}
+{% assign aria_req_objs = "" | split: "," %}
+{% for rule in site.data.wcag-act-rules.rules["act-rules"] %}
+  {% if rule.frontmatter.accessibility_requirements %}
+    {% for req in rule.frontmatter.accessibility_requirements %}
+      {% assign req_key = req[0] %}
+      {% assign req_title = req[1].title | default: req_key %}
+      {% assign req_key_prefix = req_key | slice: 0, 4 %}
+      {% unless req_key_prefix == "wcag" %}
+        {% assign obj = req_title | append: '|||' | append: req_key %}
+        {% unless aria_req_objs contains obj %}
+          {% assign aria_req_objs = aria_req_objs | push: obj %}
+        {% endunless %}
+      {% endunless %}
+    {% endfor %}
+  {% endif %}
+{% endfor %}
+{% assign aria_req_objs = aria_req_objs | sort %}
 
-<ul>
-{%- for rule in site.data.wcag-act-rules.rules["act-rules"] %}
-  {%- if rule.deprecated == true %}
-    <li><a href="{{ rule.permalink | relative_url }}">{{ rule.title | replace: 'DEPRECATED — ', '' }}</a></li>
-  {%- endif %}
-{%- endfor %}
-</ul>
+<h2 data-aria-section="true">ARIA Rules</h2>
+<div class="aria-rules-list-container">
+{% assign found_aria_rule = false %}
+{% for aria_req_obj in aria_req_objs %}
+  {% assign pair = aria_req_obj | split: '|||' %}
+  {% assign req_title = pair[0] %}
+  {% assign req_key = pair[1] %}
+  {%- comment -%} Now, list all rules for this requirement key {%- endcomment -%}
+  {% assign rules_for_req = "" | split: "," %}
+  {% for rule in site.data.wcag-act-rules.rules["act-rules"] %}
+    {% assign has_req = false %}
+    {% if rule.frontmatter.accessibility_requirements %}
+      {% for req in rule.frontmatter.accessibility_requirements %}
+        {% if req[0] == req_key %}
+          {% assign has_req = true %}
+        {% endif %}
+      {% endfor %}
+    {% endif %}
+    {% if has_req %}
+      {% assign rules_for_req = rules_for_req | push: rule %}
+    {% endif %}
+  {% endfor %}
+  {% if rules_for_req.size > 0 %}
+    {% assign comma_index = req_title | split: ',' %}
+    {% if comma_index.size > 1 %}
+      <p><strong>{{ comma_index[0] }}</strong>{{ req_title | remove_first: comma_index[0] }}</p>
```

Comment:
similarly to what we've done for the SCs,  I think we should make each requirement a heading